web/src/compose_validate.js

```diff
@@ -138,7 +155,8 @@ export function warn_if_mentioning_unsubscribed_user(mentioned) {
 
     if (needs_subscribe_warning(user_id, sub.stream_id)) {
         const $existing_invites_area = $(
-            `#compose_banners .${CSS.escape(compose_banner.CLASSNAMES.recipient_not_subscribed)}`,
+            `#compose_banners .${CSS.escape(compose_banner.CLASSNAMES.recipient_not_subscribed)},
+            .edit_form_banners .${CSS.escape(compose_banner.CLASSNAMES.recipient_not_subscribed)}`,
```

Comment:
Should we add a line here? I think we need to check all possible areas, including the `edit_form_banners`.